src/background/handlers/getCategoryTrends.js

```diff
@@ -0,0 +1,32 @@
+import { db } from '../initdb.js';
+
+export async function getCategoryTrends(days) {
+  const cutoff = Date.now() - days * 24 * 60 * 60 * 1000;
```

Comment:
`24 * 60 * 60 * 1000` needs to be a CONSTANT and it's used everywhere so let's put it in an utils.js

Reply:
or well, if all the functions are in the same module, at the top of the module